src/main/java/randoop/reflection/OperationExtractor.java

```diff
@@ -41,24 +50,51 @@
   /** The class type of the declaring class for the collected operations */
   private ClassOrInterfaceType classType;
 
+  /** The list of {@code Pattern} objects to omit matching operations */
+  private List<Pattern> omitPatterns;
+
   /**
    * Creates a visitor object that collects the {@link TypedOperation} objects corresponding to
-   * members of the class type and satisfying the given predicate.
+   * members of the class type and satisfying the given predicates.
```

Comment:
OperationExtractor documentation doesn't mention omitmethods, but the class does use omitmethods as a filter